draft-ietf-httpbis-digest-headers.md

```diff
@@ -51,13 +51,6 @@ normative:
       ins: National Institute of Standards and Technology, U.S. Department of Commerce
     date: 2008-10
     target: https://csrc.nist.gov/csrc/media/publications/fips/180/3/archive/2008-10-31/documents/fips180-3_final.pdf
-  FIPS180-4:
```

Comment:
I'm not sure if you want to remove FIPS180-4, or if you want to update references to FIPS180-3 to point to FIPS180-4. Then remove FIPS180-3.

Note that references to FIPS180-1 are due to historical reference, e.g. an algorithm that was removed from later drafts.

Reply:
Oh, I just saw https://github.com/httpwg/http-extensions/issues/832